2-mandatory/6-journey-planner.js

```diff
@@ -16,17 +16,20 @@
 
  Hint: search for string methods like Includes and IndexOf.
 */
-
 function checkCodeIsThere(stringText) {
   let magicWord = "code";
   //edit code below
   if (stringText) {
     return stringText;
+  }
+  if (stringText.includes(magicWord)) {
+    return stringText.indexOf(magicWord);
   } else {
     return "Not found";
   }
```

Comment:
You've got the right idea here, but I think you need to remove the original placeholder code! Like I explained before, something like `if` that converts a string to a boolean will always interpret the string as `true` if it has any text at all; as a result, almost everything you pass in here will return the original string and never reach the code below.
```suggestion
  if (stringText.includes(magicWord)) {
    return stringText.indexOf(magicWord);
  } else {
    return "Not found";
  }
```